Assets/Newtonsoft.Json.UnityConverters.Tests/Newtonsoft.Json.UnityConverters.Tests.asmdef

```diff
@@ -1,20 +1,40 @@
 {
     "name": "Newtonsoft.Json.UnityConverters.Tests",
+    "rootNamespace": "",
     "references": [
-        "Newtonsoft.Json.UnityConverters"
+        "Newtonsoft.Json.UnityConverters",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner"
     ],
     "includePlatforms": [],
     "excludePlatforms": [],
     "allowUnsafeCode": false,
     "overrideReferences": true,
     "precompiledReferences": [
-        "Newtonsoft.Json.dll"
+        "Newtonsoft.Json.dll",
+        "nunit.framework.dll"
     ],
     "autoReferenced": false,
     "defineConstraints": [
+        "UNITY_INCLUDE_TESTS",
```

Comment:
There seems to be an asmdef format migration to have taken place here.

This project is developed using Unity 2018 as Unity can automatically migrate older versions to newer, but cannot migrate from newer versions to older.

If I open your branch in Unity 2018 I get the following error from multiple test files:

```
error CS0246: The type or namespace name 'NUnit' could not be found (are you missing a using directive or an assembly reference?)
```

Suggest reverting all changes in this file except the newly added `versionDefines` object.